modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetDataMVCResourceCommand.java

```diff
@@ -472,7 +472,7 @@ private JSONArray _getViewURLsJSONArray(
 						currentLocale,
 						analyticsReportsInfoItem.getDefaultLocale(object))
 				).put(
-					"languageId", LocaleUtil.toBCP47LanguageId(currentLocale)
```

Comment:
Maybe this change fixes [LPS-129171 Wrong languageId in Content Performance panel for zh-CN](https://issues.liferay.com/browse/LPS-129171)?

Reply:
Yeahh, i think that :)

Reply:
I think the answer is `yes` because I see the Chinese language perfect in the screenshot ❤️